heapsort.c

```diff
@@ -0,0 +1,61 @@
+#include<stdio.h>
+#include <conio.h>
+void Adjust(int Heap_of_Numbers[],int i)  /*Function to arrange the elements in the heap*/
+{
+int j;

```

Comment:
Codacy found an issue: [The scope of the variable 'j' can be reduced.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6320511)

Reply:
j is the variable used to jump between the elements in an array